kernel/packages/scene-system/scene.system.ts

```diff
@@ -15,6 +16,19 @@ class WebWorkerScene extends SceneRuntime {
 
   devToolsAdapter?: DevToolsAdapter
 
+  updateEnabled: boolean = true
+
+  constructor(transport: ScriptingTransport) {
+    super(transport)
+
+    addEventListener('error', (e) => {
+      console.warn('🚨🚨🚨 Unhandled error in scene code. Disabling worker update loop 🚨🚨🚨')
+      console.error(e)
+      this.updateEnabled = false
+      debugger
```

Comment:
What's the behaviour of `debugger` when dev tools is closed? This wouldn't pause the execution of the client?

Reply:
> What's the behaviour of debugger when dev tools is closed?
It does nothing, that is the spec of JS, it is safe and sound

Reply:
no, it works only with the dev tools open, that's why we left it there

Reply:
love the idea of putting `debugger` in this kind of errors and those emojis! ❤️